src/Microsoft.DotNet.Build.Tasks/UpdatePackageDependencyVersion.cs

```diff
@@ -22,7 +22,7 @@ public override bool VisitPackage(JProperty package, string projectJsonPath)
             string dependencyVersion;
             if (package.Value is JObject)
             {
-                dependencyVersion = package.Value["version"].Value<string>();
+                dependencyVersion = package.Value["version"]?.Value<string>();
```

Comment:
I missed this is part of the update I thought it was part of the validate, so I'm OK with this.